lib/galaxy/tools/actions/upload_common.py

```diff
@@ -283,7 +284,7 @@ def _chown( path ):
         try:
             # get username from email/username
             pwent = trans.user.system_user_pwent(trans.app.config.real_system_username)
```

Comment:
I was wondering if its possible to use the get_destination_configuration function here. If yes: what would be the access path. Otherwise how can we find out that the variable is not defined in the config.